packages/app/src/systems/Bridge/components/NetworkConnectCard/NetworkConnectCard.tsx

```diff
@@ -0,0 +1,97 @@
+import { cssObj } from '@fuel-ui/css';
+import {
+  Card,
+  Stack,
+  Text,
+  Flex,
+  Image,
+  Button,
+  Icon,
+  Box,
+} from '@fuel-ui/react';
+import type { Bech32Address } from 'fuels';
+
+import { formatAddress } from '~/systems/Core/utils';
+
+type NetworkConnectCardProps = {
+  networkName: string;
+  networkImageUrl: string;
+  label: string;
+  isConnecting?: boolean;
+  currentAccount?: `0x${string}` | Bech32Address;
```

Comment:
can we change to `string`? otherwise it will make you need to do some typeguard before passing to this currentAccount component